src/inference/inference.ts

```diff
@@ -3,8 +3,18 @@ import {
   EmbedRequestInputsInner,
   EmbedRequestParameters,
   InferenceApi,
+  Rerank200Response as RerankResponse,
```

Comment:
This naming is kind of unfortunate, the type in the generated code. Should we maybe look into fixing the spec upstream to make this a little cleaner? I'd need to look at it but I think there's a way we've resolved some of this naming stuff by tweaking the spec. I think there may be a model that needs the x-component tag or whatever.

Just a thought.

Reply:
Good thinking -- I've opened a PR here https://github.com/pinecone-io/apis/pull/147